res/controllers/Behringer-DDM4000.midi.xml

```diff
@@ -0,0 +1,697 @@
+<?xml version='1.0' encoding='utf-8'?>
+<MixxxControllerPreset mixxxVersion="1.6.2+" schemaVersion="1">
+    <info>
+        <name>Behringer DDM4000</name>
+        <description>DDM4000 Mapping</description>
+	<author>TheArmitage</author>
+    </info>
+    <controller id="Midi-Keyboard">
+        <scriptfiles/>
+        <controls>
+            <control>
+                <group>[EqualizerRack1_[Channel1]_Effect1]</group>
+                <key>button_parameter3</key>
+                <description>MIDI Learned from 28 messages.</description>
+                <status>0x90</status>
+                <midino>0x00</midino>
+                <options>
+                    <normal/>
+                </options>
+            </control>
+            <control>
+                <group>[Master]</group>
+                <key>gain</key>
```

Comment:
Be aware that this control does actually not need to be adjusted, see https://manual.mixxx.org/2.3/en/chapters/user_interface.html#headphone-and-master-mix-controls > Master knob
(except you pipe through some quiet Aux signal or Mic signal)

In case this knob also works as hardware control for the main output it's recommended to not map its MIDI signal.